README.md

```diff
@@ -381,7 +380,7 @@ Note that BTCPayServer developers will not spend excessive time testing your ima
 | btcpayserver/litecoin | 0.21.2.1-2 | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Litecoin/0.21.2.1-2/Litecoin/0.21.2.1/linuxamd64.Dockerfile) | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Litecoin/0.21.2.1-2/Litecoin/0.21.2.1/linuxarm32v7.Dockerfile) | [✔️](https://raw.githubusercontent.com/btcpayserver/dockerfile-deps/Litecoin/0.21.2.1-2/Litecoin/0.21.2.1/linuxarm64v8.Dockerfile) | [Github](https://github.com/btcpayserver/dockerfile-deps) - [DockerHub](https://hub.docker.com/r/btcpayserver/litecoin) |
 | wakiyamap/docker-monacoin | 0.20.2 | [✔️](https://raw.githubusercontent.com/wakiyamap/docker-bitcoin/master/monacoin/0.20.2/Dockerfile) | ️❌ | ️❌ | [Github](https://github.com/wakiyamap/docker-bitcoin) - [DockerHub](https://hub.docker.com/r/wakiyamap/docker-monacoin) |
 | redis | 6.2.2-buster | [✔️](https://raw.githubusercontent.com/docker-library/redis/f1a8498333ae3ab340b5b39fbac1d7e1dc0d628c/5.0/Dockerfile) | ️❌ | ️❌ | [Github](https://github.com/docker-library/redis) - [DockerHub](https://hub.docker.com/_/redis) |
-| bluewalletorganization/lndhub | v1.4.1 | [✔️](https://raw.githubusercontent.com/BlueWallet/LndHub/v1.4.1/Dockerfile) | [✔️](https://raw.githubusercontent.com/BlueWallet/LndHub/v1.4.1/Dockerfile) | [✔️](https://raw.githubusercontent.com/BlueWallet/LndHub/v1.4.1/Dockerfile) | [Github](https://github.com/BlueWallet/LndHub) - [DockerHub](https://hub.docker.com/r/bluewalletorganization/lndhub) |
```

Comment:
This is automatically generated. Either remove this change, or find a way to make sure it doesn't generate again (`contrib/DockerFileBuildHelper\update-repo.ps1`)

Reply:
What is this